src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -390,7 +418,9 @@
 
         onDrop(event) {
             const targetContainer = event.currentTarget;
-            const dragContainerItems = targetContainer.querySelectorAll('.ibexa-collapse--field-definition');
+            const dragContainerItems = targetContainer.querySelectorAll(
```

Comment:
It looks for placeholder position here https://github.com/ibexa/admin-ui/blob/f4bd86034dcaeb3dece0daaab465e79a206aaa03/src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js#L428 , before there was never placeholder element in this array, that's why index was always -1 and new field was added at the end